### PR TITLE
batches: update documentation for Github Org with SSO enabled

### LINKED ISSUE #35071: batches: GitHub.com with SSO has special token requirements
Apparently, when using SSO with GitHub.com, you need to white-list PATs for use on the particular organization first. Otherwise, github rejects requests with a link to authorize that PAT first. 

Customer ran into this. 

More details here: https://sourcegraph.slack.com/archives/CMMTWQQ49/p1651853501985949

I think this could be as simple as a docs change, educating people about this quirk.

### PR DESCRIPTION
**Pull request by BolajiOlajide**
Closes #35071

## Test plan

Documentation update.

### CODE REVIEW

**Review by eseliger — APPROVED** [review]

File: doc/batch_changes/how-tos/configuring_credentials.md [hunk 1]
```diff
@@ -136,6 +136,8 @@ This is done by selecting the relevant checkboxes when creating the token:
 
 > NOTE: `workflow` is technically only required if your batch changes modify files in the `.github` directory of a repository, but we recommend enabling it regardless to avoid confusing errors at a later time.
 
+When working with organizations that have SAML SSO (Single Sign On) enabled, configuring credentials requires an additional step that [involves white-listing the token for use in that organization](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-a-personal-access-token-for-use-with-saml-single-sign-on).
+
```
Comment (courier-new) [inline doc/batch_changes/how-tos/configuring_credentials.md:140]: Is this only relevant to GH Enterprise, or is it a thing on Github.com, too?

Comment (adeola-ak) [reply doc/batch_changes/how-tos/configuring_credentials.md:140]: yup! its just enterprise

Comment (eseliger) [reply doc/batch_changes/how-tos/configuring_credentials.md:140]: this is enterprise _cloud_ though, so github.com I think

Comment (BolajiOlajide) [reply doc/batch_changes/how-tos/configuring_credentials.md:140]: [Github Docs](https://docs.github.com/en/enterprise-cloud@latest/organizations/managing-saml-single-sign-on-for-your-organization/enabling-and-testing-saml-single-sign-on-for-your-organization)

CleanShot 2022-06-14 at 13 15 44@2x

Comment (BolajiOlajide) [reply doc/batch_changes/how-tos/configuring_credentials.md:140]: Yup. It's for enterprise cloud. You're right @eseliger .

Comment (BolajiOlajide) [reply doc/batch_changes/how-tos/configuring_credentials.md:142]: Updated this to reflect on both `Github.com` and `Github Enterprise` - from my understanding it's applicable to both as Github enterprise users also have access to turn on SSO on their account.

CleanShot 2022-06-14 at 13 34 31@2x

**Review by courier-new — APPROVED** [review]

**Pull request merged**